### ISSUE TITLE
`pkgtools.pypi.real_name` does not work as expected

### ISSUE THREAD

**Issue by paternal** [rubik/pkgtools#6]
The [documentation](https://github.com/rubik/pkgtools/blob/17c9dc93835b01f49c4cd96725ec069c0b3db8c9/pkgtools/pypi.py#L39-L40) reads:

``` python
>>> real_name('sphinx')
'Sphinx'
```

However, I get:

``` python
>>> from pkgtools.pypi import real_name
>>> real_name("sphinx")
'sphinx'
>>> real_name("Sphinx")
'sphinx'
>>> 
```

I am on Debian testing, in a virtualenv, with:

``` sh
$ pip freeze
pkgtools==0.7.3
$ python --version
Python 3.5.1+
```

**Comment by rubik** [comment]
Yeah, that's because PyPI no longer redirects to the correct name. Now even a lowercase name works and this function has no value.
Please note that this project has been unmaintained for years. Unfortunately I don't have the time to maintain it now and almost nobody uses it anymore.

**Comment by paternal** [comment]
> Yeah, that's because PyPI no longer redirects to the correct name. Now even a lowercase name works and this function has no value.

It does not:

``` python
>>> from pkgtools.pypi import pypi_client
>>> pypi = pypi_client()
>>> pypi.package_releases('sphinx')
[]
>>> pypi.package_releases('Sphinx')
['1.4b1', '1.3.6']
```

> Please note that this project has been unmaintained for years. Unfortunately I don't have the time to maintain it now and almost nobody uses it anymore.

I suspected that it was unmaintained, but I still wanted to report it just in case.

Thanks for your work!

**Comment by rubik** [comment]
Sorry, I was being unclear. The "simple" index (https://pypi.python.org/simple) is now case-insensitive. The package endpoint is not. Since `real_name` used the simple index to find out the real name, it does not work anymore.
